metricbeat/module/system/fsstats/fsstats.go

```diff
@@ -0,0 +1,94 @@
+// +build darwin linux openbsd windows
+
+/*
+
+An example event looks as following:
+
+    {
+      "@timestamp": "2016-04-26T19:30:19.475Z",
+      "beat": {
+        "hostname": "ruflin",
+        "name": "ruflin"
+      },
+      "metricset": "filesystem",
+      "module": "system",
+      "rtt": 434,
+      "system-filesystem": {
```

Comment:
@monicasarbu That is the namespacing we require. This is always `$module-$metricset` for all events.